### DISCUSSION TITLE
How many Users are the Limit for Nextcloud AIO?

### DISCUSSION THREAD

**Discussion by smooth-baking-powder** [nextcloud/all-in-one#2615] [Questions] [answered]
Hi,

in my company a Collegue asked me (cause i use nextcloud private) Nextcloud AIO is a good choice for a Customer with ~1000 users.

Is Nextcloud AIO a good choice? In the future, the registered user can be increase.

Thanks

**Comment by szaimen** [comment, marked as answer]
Hi, in general, AIO should scale pretty well with the given hardware. So it mostly depends on the hardware how good AIO will perform: See https://github.com/nextcloud/all-in-one/discussions/1335.

BTW, for such a big instance with that many users, you should really look into getting an enterprise subscription. For that you may fill out the form at https://nextcloud.com/all-in-one
